cmd/common.go

```diff
@@ -16,6 +16,7 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/sorintlab/stolon/internal/cluster"
```

Comment:
please reorganize imports to split between stdlib, stolon packages and external packages.

Reply:
Sure. I missed it :-( . Thanks for the catch. Updated and amended the commit